src/sentry/search/events/datasets/discover.py

```diff
@@ -1346,7 +1348,17 @@ def _resolve_user_misery_function(self, args: Mapping[str, str], alias: str) ->
                             "plus",
                             [
                                 Function(
-                                    "nullIf", [Function("uniq", [self.builder.column("user")]), 0]
+                                    "nullIf",
+                                    [
+                                        Function(
+                                            "uniqIf",
+                                            [
+                                                self.builder.column("user"),
+                                                Function("greater", [column, 0]),
```

Comment:
can we just add a short comment here too about why we're doing `Function("greater", [column, 0])`